api/services/email_mailgun.js

```diff
@@ -71,5 +73,30 @@ module.exports = {
       '</a>'
 
     return link
+  },
+  submitEmail: function (data, callback) {
+    if (process.env.NODE_ENV === 'testing') {
+      return callback(undefined, 'Email sent')
+    }
+
+    var addresses = R.map(x => ({address:x}), R.keysIn(data.email))
```

Comment:
`x => ({ address: x })` could be replaced with `R.objOf('address')`. Is there a reason for the `R.keysIn` rather than `R.keys`?